### PR TITLE
[Ruby - In Review] Upgrade ruby run-time version requirement warning

### PR DESCRIPTION
**Pull request by vishrutshah**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: Samples/azure-storage/Azure.Java.Fluent/implementation/StorageAccountInner.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 import petstore.StorageAccountProperties;
 import com.microsoft.azure.Resource;
+import com.microsoft.azure.Resource;
```
Comment (veronicagg) [inline Samples/azure-storage/Azure.Java.Fluent/implementation/StorageAccountInner.java]: @jianghaolu you may want to look at why the import is repeated in the sample. This is not related to the change @vishrutshah is doing, he's just regenerating samples.

**Comment by veronicagg** [comment]
@vishrutshah change looks good to me. There was an issue with Travis, saying that curl is not installed, may need to be re-tested. Thanks!

File: Samples/azure-storage/Azure.Java.Fluent/implementation/StorageAccountInner.java (hunk 1 above)
Comment (jianghaolu) [inline Samples/azure-storage/Azure.Java.Fluent/implementation/StorageAccountInner.java]: It's a bug in Java codegen. I've already fixed it locally. I'll make sure the samples are regenerated when I send a PR. Thanks for catching this!

**Comment by vishrutshah** [comment]
Thanks for the review!!
